lua/lspconfig/server_configurations/jedi_language_server.lua

```diff
@@ -16,6 +16,7 @@ return {
     single_file_support = true,
   },
   docs = {
+    package_json = 'https://github.com/pappasam/coc-jedi/blob/main/package.json',
```

Comment:
This seems to be the web URL, here's the raw JSON one:

```suggestion
    package_json = 'https://raw.githubusercontent.com/pappasam/coc-jedi/main/package.json',
```